parlemonde/1village
# feat: add enigma category

### Motivation

To solve this ticket : https://trello.com/c/YlZP8kPE/24-rajouter-une-cat%C3%A9gorie-d%C3%A9nigme-lieu-myst%C3%A8re

### Changes

Create new category named "Lieu mystère" for enigma. Then create some subcategories 

### Test

- `git checkout feat/add-enigma-category`
- Go to Homepage
- Click on "Créer une énigme"
- You should see 4 categories instead of 3 on master branch
- Select "Lieu mystère", then click on "Créer une nouvelle énigme"
- You should see 4 subcategories :

> un monument commémoratif
> un édifice remarquable
> une salle de spectacle
> un ouvrage d'art
